packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m

```diff
@@ -103,13 +103,13 @@ - (dispatch_queue_t)methodQueue
 
   if (@available(iOS 11.0, *)) {
     NSString *dismissButtonStyle = [arguments valueForKey:@"dismissButtonStyle"];
-    if ([dismissButtonStyle isEqualToString:@"done"]) {
+    if ([@"done" isEqualToString:dismissButtonStyle]) {
       safariVC.dismissButtonStyle = SFSafariViewControllerDismissButtonStyleDone;
     }
-    else if ([dismissButtonStyle isEqualToString:@"close"]) {
+    else if ([@"close" isEqualToString:dismissButtonStyle]) {
       safariVC.dismissButtonStyle = SFSafariViewControllerDismissButtonStyleClose;
     }
-    else if ([dismissButtonStyle isEqualToString:@"cancel"]) {
+    else if (["cancel" isEqualToString:dismissButtonStyle@]) {
```

Comment:
```suggestion
    else if (["cancel" isEqualToString:dismissButtonStyle]) {
```

I'm guessing the extra `@` here was unintentional?

Also, why is it safer to compare the other way around here? 🤔

Reply:
yeah, it was, I've already corrected that.

This comparison is safer, since for constants we can bu sure its type and non nullability. Which is not the case for value read from arguments.

Reply:
Ah, I didn't think about type. Neat 👌 

For the record, doing `isEqualToString` on a null `NSString` results in null which is falsey. I made sure to test without setting a style as well.

Reply:
Great You thought about that. I assumed that @sjchmiela suggestions helps with nulls as well, great to know it is not necessary :)

Reply:
Yeah, I didn't think about the user sending in e.g. a number though ☺️